stacks/port_tcp.go

```diff
@@ -9,13 +9,23 @@ import (
 	"github.com/soypat/seqs/eth"
 )
 
+// <unhelpful and misleading !>
 // tcphandler represents a user provided function for handling incoming TCP packets on a port.
 // Incoming data is sent inside the `pkt` TCPPacket argument when pkt.HasPacket returns true.
 // Outgoing data is stored into the `response` byte slice. The function must return the number of
 // bytes written to `response` and an error.
-//
+// </unhelpful and misleading !>
+
+// <suggeted replacement>
+// tcphandler represents a user provided function for handling incoming TCP packets on a port.
+// Incoming data is passed in a 'pkt' to the recv function which is invoked whenever data arrives (by RecvEth)
+// Outgoing data is written into the `dst` byte slice (from the tx ring buffer). The function must return the number of
+// bytes written to `response` and an error.
+// TCPConn provides an implemntation of this interface - note .send is ONLY called by HandleEth
+// </suggeted replacement>
+
 // See [PortStack] for information on how to use this function and other port handlers.
-type itcphandler interface {
+type itcphandler interface {  //note TCPConn is our implementation of this interface 
```

Comment:
good note. Include it in itcphandler documentation by putting it above the interface declaration.